packages/core/src/utils/elementReady.ts

```diff
@@ -15,9 +17,19 @@ export const ready = function (name: string, callback?: ReadyHandler): void {
     try {
       callbackCollection.forEach((callback) => callback && callback());
     } catch (e) {
-      setTimeout(() => {
-        throw e;
-      });
+      if (e instanceof DOMException) {
+        const isCustomElementRegistryError =
+          e.name === 'NotSupportedError' && e.message.includes('CustomElementRegistry');
+        if ((isLocalhost && isCustomElementRegistryError) || !isCustomElementRegistryError) {
+          setTimeout(() => {
+            throw e;
+          });
+        }
+      } else {
+        setTimeout(() => {
+          throw e;
+        });
+      }
```

Comment:
```suggestion
      if (!(isLocalhost && e instanceof DOMException &&
        e.name === 'NotSupportedError' &&
        e.message.includes('CustomElementRegistry'))) {
          setTimeout(() => {
            throw e;
          });
      }
```

Reply:
one more thing, have you confirmed that the error message for duplicate custom elements of all 3 major browsers contain 'CustomElementRegistry'?